docs/source/recipes/librispeech/conformer_ctc.rst

```diff
@@ -465,7 +465,7 @@ After downloading, you will have the following files:
   |   `-- pretrained.pt
   |-- log
   |   `-- log-decode-2021-11-09-17-38-28
-  `-- test_wavs
+  `-- test_waves
```

Comment:
This one is also correct. please see
https://huggingface.co/csukuangfj/icefall-asr-librispeech-conformer-ctc-jit-bpe-500-2021-11-09/tree/main/test_wavs

Reply:
After confirming that you are correct, in addition, I also confirmed the file path to timit and found the same to be test_wavs, meaning that neither file needs to be modified. But I would suggest that it would be great if the path format could be standardised.

Reply:
> This one is also correct. please see https://huggingface.co/csukuangfj/icefall-asr-librispeech-conformer-ctc-jit-bpe-500-2021-11-09/tree/main/test_wavs

Could you fix this one?

Reply:
> 



> > This one is also correct. please see https://huggingface.co/csukuangfj/icefall-asr-librispeech-conformer-ctc-jit-bpe-500-2021-11-09/tree/main/test_wavs
> 
> Could you fix this one?

Is it currently reverting from test_waves to test_wavs? In that case, I need to restore the file paths in both the librispeech and timit folders to test_wavs.